### DISCUSSION TITLE
Certificate automation

### DISCUSSION THREAD

**Discussion by pilif** [blinksh/blink#1537] [Q&A]
Blink is the only iOS SSH client with support for certificate authentication and it’s the only viable ssh client for me because of that. 

However, as our certificates have a very short expiration time, I would love if there was some way to automate that process at least a bit. 

Is there a blink URL scheme by which I can add a certificate to a public key? That would be very useful already. 

Or alternatively, is there a way to link a certificate to iCloud Drive rather than importing it one-time

**Comment by yury** [comment]
I think, this is really good candidate for use of new [AppIntents](https://developer.apple.com/documentation/AppIntents) swift framework.

**Reply by pilif** [reply]
TBH, a url would be preferable for me because I could easily update the code to sign the certs to provide a link on a webpage to add it to blink

**Comment by carloscabanero** [comment]
Created #1909 to track there
